Seminar2/Task2.py

```diff
@@ -0,0 +1,23 @@
+#Напишите программу, которая получает целое число и возвращает его 
+#шестнадцатеричное строковое представление. Функцию hex используйте для проверки своего результата.
+
+num = 189
+def func_hex(num):
+    result = []
+    letters = ['a', 'b', 'c', 'd', 'i', 'f']
```

Comment:
А почему не добавили все цифры из системы? Или использовать строку `0123456789abcdef`, то остальной код будет работать так же, без изменений смотрите ниже
```suggestion
    letters = '0123456789abcdef'
```